### PR TITLE
Booksearch

### PR DESCRIPTION
**Pull request by pyccia**

### CODE REVIEW

File: src/components/booksearch/booksearch.css [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    padding-right: 10%;
+    color: #5e5e5e;
+    font-weight: 6;
+    }
+  
+
+
+.booksearch__backgroundandscroll {
+height: 100vh;
+width: 100vw;
```
Comment (KubaC701) [inline src/components/booksearch/booksearch.css]: you don't need to specify it. Container will adjust to the height of the element and fill the 100% of the screen width

File: src/components/booksearch/BookSearch.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import './booksearch.css';
+import books from '../../data/books.json';
+
+const BookSearch = () => {
+  return (
+   
+   
+   
+   
+   <main  className="booksearch__backgroundandscroll">
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]: this class should be `booksearch` as it is the main block of the page

File: src/components/booksearch/BookSearch.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+
+
+
+
+    <div className="booksearch">
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]:
```suggestion
    <div className="booksearch__container">
```
maybe?

File: src/components/booksearch/BookSearch.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+      <div >
+        <h1>logo&apos;s space</h1>
+      </div>
+      <div className="booksearch__searcher">
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]:
```suggestion
      <div className="booksearch__search">
```
I think `search` is descriptive enough

File: src/components/booksearch/BookSearch.jsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        <input className="booksearch__input" type="text" name="searcher" id="searcher" placeholder="Book's title" />
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]: the same here

File: src/components/booksearch/BookSearch.jsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+
+        <button type="submit" className="booksearch__searchbutton">
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]: if you want to use 2 words as a className separate words by `-`
```suggestion
        <button type="submit" className="booksearch__search-button">
```

File: src/components/booksearch/BookSearch.jsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+
+
+
+
+
+
+      <div className="booksearch__items">
+        {books.map((book) => (
+          <div className="booksearch__item" type="submit" key={`${book.title} ${book.author}`}>
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]: `type="submit` is not valid for `div`

File: src/components/booksearch/BookSearch.jsx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+            <div>
+            <img src={book.thumbnail} alt={book.title} className="booksearch__img" />
```
Comment (KubaC701) [inline src/components/booksearch/BookSearch.jsx]: maybe `booksearch__thumbnail` would be better?

File: src/components/booksearch/booksearch.css [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+span {    
+    font-size: 12px;
+    text-align: center;
+    padding-top: 10px;  
+    padding-bottom: 30px;
+    padding-left: 10%;
+    padding-right: 10%;
+    color: #5e5e5e;
+    font-weight: 6;
+    }
```
Comment (KubaC701) [inline src/components/booksearch/booksearch.css]: don't style by tag. It'll leak to the whole app

File: src/components/booksearch/booksearch.css [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+        background-color: #ffffff;
+        color: #000000;
+        border-radius: 8px;
+        padding: 8px;
+        border: none;
+        margin-bottom: 10px;
+      }
+      .booksearch__searchbutton {
+        background-color: #E3893E;
+        transition: background-color 0.2s ease-in-out;
```
Comment (KubaC701) [inline src/components/booksearch/booksearch.css]: nice transition effect

**Review by KubaC701 — APPROVED** [review]

File: src/components/booksearch/BookSearch.jsx (hunk 2 above)
Comment (pyccia) [reply src/components/booksearch/BookSearch.jsx]: I've just removed the background. I don't need this class anymore

**Pull request merged**